src/Domain/Donation.php

```diff
@@ -390,6 +402,7 @@ public function toHookModel(): array
         }
 
         $data['updatedTime'] = $this->getUpdatedDate()->format(DateTimeInterface::ATOM);
+        $data['refundedTime'] = $this->refundedAt?->format(DateTimeInterface::ATOM);
```

Comment:
We could just comment out this line until SF is ready to recieve the refund timestamps so that this can be merged, and will start recording donation times to the MySQL db.